cmd/workflow-controller/main.go

```diff
@@ -81,11 +81,7 @@ func NewRootCommand() *cobra.Command {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
-			wfController.UpdateConfig()
```

Comment:
The Config is now updated inside the `NewWorkflowController` constructor. The Controller should have the responsibility to update itself... it should not be left up to the caller.